presto-iceberg/src/main/java/com/facebook/presto/iceberg/TableStatisticsMaker.java

```diff
@@ -322,11 +329,16 @@ public void updateColumnSizes(Partition summary, Map<Integer, Long> addedColumnS
         }
         for (Types.NestedField column : summary.getNonPartitionPrimitiveColumns()) {
             int id = column.fieldId();
-
-            Long addedSize = addedColumnSizes.get(id);
-            if (addedSize != null) {
-                columnSizes.put(id, addedSize + columnSizes.getOrDefault(id, 0L));
+            com.facebook.presto.common.type.Type type = toPrestoType(column.type(), typeManager);
+            if (type instanceof FixedWidthType) {
+                continue;
```

Comment:
Can you add a note explaining why we're doing this

Reply:
fixed